lang/python/python3-jinja2/Makefile

```diff
@@ -0,0 +1,44 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-jinja2
```

Comment:
Why? This is already here:
https://github.com/openwrt/packages/blob/master/lang/python/Jinja2/Makefile